- Recognize PT_OPENBSD_MUTABLE with LLVM's readobj / objdump.
- Implement support for PT_OPENBSD_NOBTCFI in lld(1).  This can be set using
  the -z nobtcfi option.

Index: llvm/tools/llvm-readobj/ELFDumper.cpp
--- llvm/tools/llvm-readobj/ELFDumper.cpp.orig
+++ llvm/tools/llvm-readobj/ELFDumper.cpp
@@ -1369,8 +1369,10 @@ static StringRef segmentTypeToString(unsigned Arch, un
     LLVM_READOBJ_ENUM_CASE(ELF, PT_GNU_RELRO);
     LLVM_READOBJ_ENUM_CASE(ELF, PT_GNU_PROPERTY);
 
+    LLVM_READOBJ_ENUM_CASE(ELF, PT_OPENBSD_MUTABLE);
     LLVM_READOBJ_ENUM_CASE(ELF, PT_OPENBSD_RANDOMIZE);
     LLVM_READOBJ_ENUM_CASE(ELF, PT_OPENBSD_WXNEEDED);
+    LLVM_READOBJ_ENUM_CASE(ELF, PT_OPENBSD_NOBTCFI);
     LLVM_READOBJ_ENUM_CASE(ELF, PT_OPENBSD_BOOTDATA);
   default:
     return "";
